esmvalcore/preprocessor/_volume.py

```diff
@@ -260,12 +260,13 @@ def volume_statistics(
                 layer_vol = np.ma.masked_where(
                     cube[time_itr, z_itr].data.mask,
                     grid_volume[time_itr, z_itr]).sum()
-
             except AttributeError:
                 # ####
                 # No mask in the cube data.
                 layer_vol = grid_volume.sum()
             depth_volume.append(layer_vol)
+
+        column = np.ma.masked_invalid(column)
```

Comment:
What is the relation of this to this PR? It seems rather unrelated to the fx stuff. @ledm, maybe you want to comment?

Reply:
This is a issue that arose now that we finally have working FX files. It is related to the issue, promise!

Reply:
I'd ask you to be a bit more specific, but really this whole function is in need of a rewrite, so until then knock yourself out.